src/core/sys/posix/signal.d

```diff
@@ -544,6 +544,28 @@ version( CRuntime_Glibc )
         void function() sa_restorer;
     }
 }
+version( CRuntime_Musl )
```

Comment:
You missed an else.

Causing: ``src/core/sys/posix/signal.d(686): Error: static assert  "Architecture not supported."``.